angular.json

```diff
@@ -13,6 +13,7 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
+            "aot": true,
```

Comment:
Could you explain why we need this change? Isn't something done by Angular by default? - https://angular.io/guide/aot-compiler

Reply:
Indeed, this is a change introduced in Angular 9. I suppose though that without this flag set, the value would be `false`, which would decrease performance (or actually not increase it even though it could) as then Angular will perform `Just-In-Time` compilation, which was default one earlier.

Reply:
But it seems like `aot` is enabled by default by Angular 9+ itself - 

>Ahead-of-Time (AOT), which compiles your app and libraries at build time. This is the default since Angular 9.

So there is probably no need including it explicitly. On the other hand, it won't hurt defining it even when it's `true` by default.